app/Http/Controllers/Auth/LoginController.php

```diff
@@ -27,13 +32,31 @@ class LoginController extends Controller
      */
     protected $redirectTo = '/';
 
+    private $employeeRepository;
+
     /**
      * Create a new controller instance.
      *
      * @return void
      */
-    public function __construct()
+    public function __construct(EmployeeRepository $employeeRepository)
     {
+        $this->employeeRepository = $employeeRepository;
```

Comment:
I believe it was there from a previous version of the __construct function, but it has been removed.